tests/test_geometry_loader.py

```diff
@@ -19,6 +16,10 @@
 from geometry_loader import GeometryLoader
 
 
+def eq_(a, b):
+    assert a == b
+
+
```

Comment:
I won't point all of these out but I'm pretty sure you can get rid of all of them.